docs/book/src/user/quick-start.md

```diff
@@ -209,8 +232,19 @@ clusterctl init --infrastructure azure
 {{#/tab }}
 {{#tab Docker}}
 
-If you are planning to use to test locally Cluster API using the Docker infrastructure provider, please follow additional
-steps described in the [developer instruction][docker-provider] page.
+<aside class="note warning">
```

Comment:
As I was going through this Quick Start, I noticed that I needed to do the setup specified in the _clusterctl for Developers_ section first. 
Maybe we should include a note here to go to `clusterctl for Developers` first in order to setup capd related artifacts.

However, if we plan on releasing artifacts like the docker images and the `infrastructure-components-development.yaml` then we can ignore this note, since everything should work automatically.

Reply:
This is not true, please see https://github.com/kubernetes-sigs/cluster-api/pull/3514#issuecomment-679933580